core-plugins/src/main/java/co/cask/hydrator/plugin/batch/source/TimePartitionedFileSetSource.java

```diff
@@ -99,12 +99,7 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
 
   @Override
   public final void prepareRun(BatchSourceContext context) {
-    Map<String, String> runtimeArgs = context.getRuntimeArguments();
     long runtime = context.getLogicalStartTime();
-    if (runtimeArgs.containsKey("runtime")) {
-      runtime = Long.parseLong(runtimeArgs.get("runtime"));
```

Comment:
was this a hack before ? but seems we we used this as a way to process for a particular time, but now that has to be passed by reconfiguring the pipeline?